beta/buster/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:buster-slim
+FROM debian:bullseye-slim
```

Comment:
Should the file be renamed too, to match this new image?

Reply:
Good point :)